pkg/server/types.go

```diff
@@ -99,13 +99,18 @@ func Wait(ctx context.Context, config *rest.Config) error {
 	return nil
 }
 
-func (s *HarvesterServer) Start() error {
+func (s *HarvesterServer) Start(listenOpts *dynamiclistener.Config) error {
 	opts := &server.ListenOpts{
 		Secrets: s.steve.Controllers.Core.Secret(),
 		TLSListenerConfig: dynamiclistener.Config{
 			CloseConnOnCertChange: true,
 		},
 	}
+
+	if listenOpts != nil {
```

Comment:
What's the reason for this change?

Reply:
the default `CloseConnOnCertChange: true` will close the API connection of the integration test since the cert is updating...